src/arrayMethodSort.js

```diff
@@ -5,7 +5,36 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let toRepeat = true;
+
+    const bubbleSort = () => {
+      const defaultCheck = (a, b) => {
+        if (String(a) > String(b)) {
+          return 1;
+        }
+
+        return -1;
+      };
+
+      const checkCondition = compareFunction || defaultCheck;
```

Comment:
We can avoid this line of code by adding  `defaultCheck` as a default parameter to `sort2 `method
Like this `function(compareFunction = defaultCheck)`